File: CodeWars/JS/7kyu_greet_me.js
// https://www.codewars.com/kata/535474308bb336c9980006f2

var greet = function (name='name') {
    return `Hello ${name[0].toUpperCase()}${name.slice(1).toLowerCase()}!`
};
console.log(greet("riley")); // 'Hello Riley!');

File: CodeWars/python/7kyu_greet_me.py
# https://www.codewars.com/kata/535474308bb336c9980006f2


def greet(name='name'):
    return f"Hello {name.lower().title()}!"


print(greet('riley'))  # 'Hello Riley!')
print(greet('molly'))  # "Hello Molly!")
print(greet('BILLY'))  # "Hello Billy!")
